webapp/src/components/sidebar_right/sidebar_right.jsx

```diff
@@ -217,7 +217,9 @@ export default class SidebarRight extends React.PureComponent {
                 >
                     <div className='todolist-header'>
                         <MenuWrapper>
-                            <button style={style.todoHeader}>
+                            <button
+                                className='todolist-header__dropdown'
```

Comment:
Since `style.todoHeader` isn't referenced anywhere, can we remove it from the `style` object?